lua/nvim-lsp-installer/servers/omnisharp/init.lua

```diff
@@ -7,55 +7,83 @@ local std = require "nvim-lsp-installer.core.managers.std"
 
 local coalesce, when = functional.coalesce, functional.when
 
+function GenerateCmd(mono_path, root_dir)
```

Comment:
Without the `local` keyword, it becomes a global!

```suggestion
local function generate_cmd(mono_path, root_dir)
```